api/v2.0/swagger.yaml

```diff
@@ -615,6 +615,68 @@ paths:
           $ref: '#/responses/401'
         '403':
           $ref: '#/responses/403'
+  /projects/{project_name}/logs:
+    get:
+      summary: Get recent logs of the projects
+      description: Get recent logs of the projects
+      tags:
+        - project
+      operationId: getLogs
+      parameters:
+        - $ref: '#/parameters/projectName'
+        - $ref: '#/parameters/requestId'
+        - $ref: '#/parameters/page'
+        - $ref: '#/parameters/pageSize'
+        - name: username
```

Comment:
Use the new query instead?

Reply:
it this like audit log? not just push, delete, also the pull event will be recorded? calling event would be more appropriate.

Reply:
update to Q in query

Reply:
So the query strings username/resource/resource_type/etc can be removed?